charts/redpanda/templates/_helpers.tpl

```diff
@@ -760,9 +656,13 @@ return licenseSecretRef.key checks deprecated values entry if current values emp
       {{- $r :=  set $ "tempCert" ( dict "name" $name "cert" $cert ) }}
 - name: redpanda-{{ $name }}-cert
   secret:
-    secretName: {{ template "cert-secret-name" $r }}
+    secretName: {{ template "redpanda.fullname" $ }}-{{ $name }}-cert
```

Comment:
Why the template for the secret name is not used here?

Reply:
I don't know.. probably got lost in the shuffle. Fixed.